components/forms/farmer-journal/ChickenFeeding.tsx

```diff
@@ -1,18 +1,26 @@
 import useJournalStatistics from "@/hooks/useJournalStatistics";
-import Label from "./Label";
+import { useFormContext } from "react-hook-form";
+import Label from "../Label";
+import PreviewField from "../PreviewField";
 import NumberInput from "./NumberInput";
-import PreviewField from "./PreviewField";
+import { FarmerJournalSchema } from "./schema";
+import ErrorMessage from "../ErrorMeesge";
 
 export default function ChickenFeeding() {
-  const { feedPerBirdInGrams, closingStock } = useJournalStatistics();
+  const { watch, formState: {errors} } = useFormContext<FarmerJournalSchema>();
+  const [inital, mortality, proplapse, feed] = watch(["fieldInitialstock", "fieldMortality", "fieldMortalityProlapse_", "fieldGivenFeed"])
+  const closingStock = inital - mortality - proplapse;
+  const feedPerBirds = closingStock <= 0 ? undefined : 
+    Number(Math.round(+`${feed * 1000 / closingStock}e2`) + "e-2")
```

Comment:
Can we make this a helper function?

```suggestion
    customRound(feed * 1000 / closingStock, 2)
```

Reply:
There is a native web API to [format numbers]([https://www.intl-explorer.com/NumberFormat?locale=en-US]) based on language, I will replace that in #4